consensus/broadcast/shardChainMessenger.go

```diff
@@ -205,6 +227,29 @@ func (scm *shardChainMessenger) PrepareBroadcastHeaderValidator(
 	}
 }
 
+func (scm *shardChainMessenger) PrepareBroadcastBlockDataWithEquivalentProofs(
+	header data.HeaderHandler,
+	miniBlocks map[uint32][]byte,
+	transactions map[string][][]byte,
+	pkBytes []byte,
+) {
+	if len(miniBlocks) == 0 {
+		return
+	}
+	dtb, err := scm.prepareDataToBroadcast(header, miniBlocks, transactions, pkBytes)
+	if err != nil {
+		return
+	}
+	// everyone broadcasts as if they were the leader
+	err = scm.delayedBlockBroadcaster.SetLeaderData(dtb.delayedBroadcastData)
+	if err != nil {
+		return
+	}
+
+	// todo: consider moving this to the initial block broadcast - optimization
```

Comment:
```suggestion
	// TODO: consider moving this to the initial block broadcast - optimization
```
use upper case to match common grep pattern?